src/interpreter/plugin/NumericAggregationPlugin.ts

```diff
@@ -91,6 +108,36 @@ export class NumericAggregationPlugin extends FunctionPlugin {
     'AVERAGEA': {
       method: 'averagea',
     },
+    'PRODUCT': {
+      method: 'product',
+    },
+    'VAR.S': {
+      method: 'vars',
+    },
+    'VAR.P': {
+      method: 'varp',
+    },
```

Comment:
Would be a valid solution to make VARP and VARS aliases here that are linking to the same method but under different name?

```suggestion
    'VAR.S': {
      method: 'vars',
    },
    'VAR.P': {
      method: 'varp',
    },
    'VARS': {
      method: 'vars',
    },
    'VARP': {
      method: 'varp',
    },
```

BTW. Are we missing arguments definitions or they are redundant?

Reply:
those functions are not using the mechanism that requires args definitions

Reply:
There is no such method as 'VARS' -- VAR should be an alias of VAR.S

Reply:
Ok, we have a problem on our list
https://github.com/handsontable/formula-parser/blob/d056ea2351153b77831b9a92842ef198b3f517f0/src/supported-formulas.js#L371-L376